retrieverdash/tests/sample_dataset.json

```diff
@@ -0,0 +1,171 @@
+{
```

Comment:
This looks like it is installing the original data. In that case, why don't you use retriever to install the original data. I may be missing something.

Reply:
Retriever is being used for installing the `original` as well as the `modified` data.